css/responsive/desktop.styl

```diff
@@ -50,22 +50,25 @@ input, select, button, .button, textarea, input[type="checkbox"], input[type=rad
 
 .single-product-image
   max-height: calc('100vh - ' + ($header-height + $footer-height))
-  
+
 .variations_form
-  .variations 
-    td 
+  .variations
+    td
       padding-bottom: $margin-small
   td.label
     padding-right: $margin-small
 
 .amount
   margin-bottom: $margin-small
-  
+
 .quantity
   margin-bottom: $margin-small
-  
+
+#product-gallery-container
+  height: calc('100vh - ' + ($header-height + $footer-height))
```

Comment:
if this is the same value as `.single-product-image` why not use a general class for that situation?